resources/user-agents/browsers/k-ninja/k-ninja-generic.json

```diff
@@ -19,9 +19,9 @@
           "match": "Mozilla/5.0 (#PLATFORM#) Gecko/* K-Ninja/*",
           "platforms": [
             "WinXPa_32",
+            "Win98_2", "Win98_3",
             "Windows",
-            "Win98_3",
-            "Win98_2", "Win98"
+            "Win98"
```

Comment:
Shouldn't `Win98` go up with the other two, and `Windows` be last?

Reply:
No. `Win98` matches to `*Win98*` while `Windows` matches to `*Windows*`.